src/Antiforgery/src/Microsoft.AspNetCore.Antiforgery.csproj

```diff
@@ -6,6 +6,9 @@
     <IsAspNetCoreApp>true</IsAspNetCoreApp>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;antiforgery</PackageTags>
+
+    <!-- Disable resx generation via Arcade due to https://github.com/dotnet/arcade/issues/3244 -->
```

Comment:
This issue is closed. If the problem is actually resolved this exception should be remove, if it's not fixed the issue should be re-opened.

Reply:
The issue has been fixed in Arcade but we don't yet have a new enough version yet. Need to update when we get a new version but don't want to block the PR on it. Will follow up.

Reply:
Actually we got new arcade since this PR was opened so we can probably get rid of these too 😄